sqlstore/src/main/resources/db/migration/V0320__auto_main.sql

```diff
@@ -81,7 +80,7 @@ CREATE TABLE PlatformSizes (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 INSERT INTO PlatformSizes(platform_platformId, partitionSize)
-  SELECT DISTINCT platform, COUNT(*) AS c FROM SequencerPartitionContainer JOIN SequencerPartitionContainer_Partition ON containerId = container_containerId GROUP BY containerId
+  SELECT platform, c FROM (SELECT DISTINCT platform, COUNT(*) AS c FROM SequencerPartitionContainer JOIN SequencerPartitionContainer_Partition ON containerId = container_containerId GROUP BY containerId) as p where platform > 0
```

Comment:
If the platform is 0, the database is wrong and should be fixed. How is this possible? You have containers with no associated platform? Are they attached to a sequencer?

Reply:
I've checked this and these containers can be deleted as they're duplicates.